includes/blocks/course-list/class-sensei-course-list-block.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * File containing extra functionalities of extended Sensei_Course_List_Block class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_Course_List_Block
+ */
+class Sensei_Course_List_Block {
+
+	/**
+	 * Sensei_Course_List_Block constructor.
+	 */
+	public function __construct() {
+		add_filter( 'render_block', [ $this, 'render_login_form_if_applicable' ], 10, 2 );
+	}
+
+	/**
+	 * Replaces content of Course List block when Show Login Form is set and user is logged out.
```

Comment:
I think we may have removed the "Show Login Form" setting?

Reply:
Thanks! Updated the comment [here](https://github.com/Automattic/sensei/pull/6343/commits/80deb082e80904abee15dfcee5af7d2b6c40002b)